### ISSUE TITLE
no way to order relationship amount types

### ISSUE THREAD

**Issue by kgeis** [kuali/research-coi#12]
Relationship amount types are shown in the order they are returned from the database. Administrators need to be able to control the order of these options.

![image](https://cloud.githubusercontent.com/assets/2237299/11548598/fc9d2bde-9912-11e5-8818-bc86f7f3693d.png)

**Comment by jflach** [comment]
Hi Ken, I cannot this reproduce it now.  Providing a way to move the order around would be an enhancement, but the system should be returning the values in the order they were in when saved. We did have a problem with that at one point.  It might have been fixed with  https://github.com/KualiCo/research-coi/pull/262.  Thanks, Jen

**Comment by kgeis** [comment]
I cannot review that changeset because it is not publicly visible.

**Comment by kgeis** [comment]
Just showing these in the order they were created is not acceptable. Even if they started in a good order, imagine I need to split "$100,000 - $149,999" in two. I would de-activate that option, and the new options, "$100,000 - $124,999" and "$125,000 - $149,999", would show out of place at the bottom of the list.

**Comment by jflach** [comment]
Yes, we would like to enhance this in future so it is not so much work to edit these lists; however, we have higher priorities at this time. Thanks.
